freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -132,7 +132,10 @@ def _handle_auto_rebuild(self, db_event):
         original_digests_by_nvr = {}
         original_nvrs_by_digest = {}
         for nvr in original_nvrs:
-            digest = self._pyxis.get_manifest_list_digest_by_nvr(nvr)
+            # Don't require that the manifest list digest be published in this case because
+            # there's a delay from after an advisory is shipped and when the published repositories
+            # entry is populated
+            digest = self._pyxis.get_manifest_list_digest_by_nvr(nvr, must_be_published=False)
```

Comment:
I believe we should `must_be_published=False` for rebuilt nvrs, not original nvrs, right?

Reply:
You are right. I misread the code.